kymatio/scattering1d/core/scattering1d.py

```diff
@@ -64,7 +68,7 @@ def scattering1d(x, pad, unpad, backend, J, psi1, psi2, phi, pad_left=0,
 
     # S is simply a dictionary if we do not perform the averaging...
     batch_size = x.shape[0]
-    kJ = max(J - oversampling, 0)
+    kJ = max(log2_T - oversampling, 0)
```

Comment:
not directly related to this PR but this note is to point out that `kJ` means basically nothing to an uninformed reader. Should keep these to a minimum. 

I get from the docstring that it has something to do with determining the cropping at the end, but it would be great if this were clearer

Reply:
Agreed